pkg/build/builder/sti.go

```diff
@@ -262,27 +261,27 @@ func (s *S2IBuilder) Build() error {
 			dockercfg.PushAuthType,
 		)
 		if authPresent {
-			glog.Infof("Using provided push secret for pushing %s image", pushTag)
+			glog.V(2).Infof("Using provided push secret for pushing %s image", pushTag)
 		} else {
-			glog.Infof("No push secret provided")
+			glog.V(2).Infof("No push secret provided")
 		}
-		glog.Infof("Pushing %s image ...", pushTag)
+		glog.V(0).Infof("Pushing %s image ...", pushTag)
```

Comment:
Isn't 1 already always shown?  I was under the impression we were using 0 for warning/erring, and 1 for basic info (still always shown by default).